custom_components/google_home/api.py

```diff
@@ -332,33 +333,34 @@ async def update_do_not_disturb(
 
         data = None
 
-        if enable is None:
+        if enable is not None:
```

Comment:
This could just be `if enable`, right? :)

Reply:
I tried that, but if the value is false, python just moves to else instead. 😊

Reply:
hm... but do we expect `True` here? Maybe then `if enable is True`? 😅

Reply:
We expected True or False if we want to set the status on the device or None if we just want get status from device 😊

Reply:
ouuu gotcha now. Thanks for explaining 👍

Reply:
No problem 😊